src/systemcmds/pwm/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ px4_add_module(
 	MAIN pwm
 	COMPILE_FLAGS
 		-Wno-array-bounds
+		-O0
```

Comment:
Why? Or is it going to be removed?

Reply:
For development, this isn't fully functional yet.

Reply:
Removed.